website/docs/foundations/breakpoints/partials/code/how-to-use.md

```diff
@@ -0,0 +1,129 @@
+!!!
+
+Once the file is imported, you will have access to the Sass helpers described below.
+
+#### Mixins
+
+Sass mixins offer the simplest, most efficient way to implement responsive layouts using the standard breakpoints defined in our system. We provide three different mixins:
+
+- `hds-breakpoint-above($name)` - for media queries that apply to a viewport range **above** a certain breakpoint (useful for a mobile-first approach)
+- `hds-breakpoint-below($name)` - for media queries that apply to a viewport range **below** a certain breakpoint (useful for a desktop-first approach)
+- `hds-breakpoint-between($lower, $upper)` - for media queries that apply to a viewport range **between** two breakpoints
+
+Here is an example of how the mixins could be used in a mobile-first responsive layout:
+
+```scss
+// from @hashicorp/design-system-components package
+@use "mixins/breakpoints" as *;
+
+.my-responsive-layout {
+  // this is the default behavior, that covers the range 0px–767px
+  display: flex;
+  direction: column;
+  gap: 1rem;
+
+  // at 768px the layout changes, from a vertical stack to an horizontal one
+  @include hds-breakpoint-above('md') {
+    direction: row;
+  }
+
+  // at larger viewports we increase the spacing between the items
+  @include hds-breakpoint-above('xl') {
+    gap: 2rem;
+  }
+}
+```
+
+The same layout could be achieved using a desktop-first approach in a similar way:
+
+```scss
+// from @hashicorp/design-system-components package
+@use "mixins/breakpoints" as *;
+
+.my-responsive-layout {
+  // this is the default behavior
+  display: flex;
+  direction: row;
+  gap: 2rem;
+
+  // at smaller viewports we decrease the spacing between the items
+  @include hds-breakpoint-below('xl') {
+    gap: 1rem;
+  }
+
+  // at 768px the layout changes, from an horizontal stack to a vertical one
+  @include hds-breakpoint-below('md') {
+    direction: column;
+  }
+}
+```
+
+Of course, these are oversimplified examples. In your implementation, you will have to choose which of the provided mixins are better suited to obtain the desired responsive layout, depending on the design specifications, the context of where the layout lives in the page, and how it relates with the other UI elements.
+
+#### Key/Value Map
+
+For special use cases, we also provide a Sass map–`$hds-breakpoints`–which is a lower level helper than the mixins. Here's an example of how this map could be used:
+
+```scss
+// explicit import of `map` module (required by Sass)
+@use "sass:map";
+// from @hashicorp/design-system-components package
+@use "mixins/breakpoints" as *;
+
+@each $name, $size in $hds-breakpoints {
+  .my-custom-breakpoint-class--#{$name} {
+    // here you have access to the breakpoint name and its value (width in px)
+  }
+}
+```
+
+### JavaScript
+
+We provide also helpers in case you need to access the breakpoints names/values in JavaScript code.
+
+You have access to the list of breakpoints' names using the `hdsBreakpointsNames` array:
```

Comment:
```suggestion
You have access to the list of breakpoint names using the `hdsBreakpointsNames` array:
```

Reply:
Related, I think we may want to remove the "s" from "breakpoints" in the variable names: https://github.com/hashicorp/design-system/pull/2848/files#r2085611901

Reply:
Done.